react-basics/meeting-notes/2022-01/JS topics to read

```diff
@@ -0,0 +1,5 @@
+Topics:
+
+*]Class vs Prototypes
```

Comment:
@dhanush027 Can we change these to simple bullet points please?
These don't render great in markdown.